### DISCUSSION TITLE
Can we order dependents by downloads?

### DISCUSSION THREAD

**Discussion by jasonwilliams** [npm/feedback#83] [Website] [answered]
This is semi-related to https://github.com/npm/feedback/discussions/82

If there’s a vulnerability or a very old dependency on npm I want to see what other libraries are using it starting with the most popular. This helps to see where it has the biggest impact and where updating is most important.

I don’t think dependents has any sort of order to it, can an option be added to start from the most downloaded downwards? Or have that as a default?

**Comment by jasonwilliams** [comment, marked as answer]
Hmm I think this may be what @bnb is asking for in https://github.com/npm/feedback/discussions/72

**Reply by MylesBorins** [reply]
Let's close this as a dupe. If you disagree feel free to mark as unanswered
